web_remember_tree_column_width/static/src/js/list_renderer.esm.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import {ListRenderer} from "@web/views/list/list_renderer";
+import {browser} from "@web/core/browser/browser";
+import {patch} from "web.utils";
+
+patch(ListRenderer.prototype, "web_remember_tree_column_width.ListRenderer", {
+    /**
+     * @override
+     */
+    computeColumnWidthsFromContent() {
+        const columnWidths = this._super.apply(this, arguments);
+        const table = this.tableRef.el;
+        const thElements = [...table.querySelectorAll("thead th:not(.o_list_button)")];
+        const self = this;
+        thElements.forEach(function (el, elIndex) {
```

Comment:
For this line, we could use the lambda function, we can improve like this

`thElements.forEach((el, elIndex) => {`

Then no need to assign self=this anymore, we can re-use `this`